debian-loongarch64.Dockerfile

```
@@ -0,0 +1,45 @@
+  CARGO_HOME=/usr/local/cargo \
+  PATH=/usr/local/cargo/bin:$PATH \
+  CC=clang \
+  CC_loongarch64_unknown_linux_gnu=clang \
+  CXX=clang++ \
+  CXX_loongarch64_unknown_linux_gnu=clang++ \
+  CFLAGS="--sysroot=/usr/loongarch64-unknown-linux-gnu/loongarch64-unknown-linux-gnu/sysroot" \
+  CXXFLAGS="--sysroot=/usr/loongarch64-unknown-linux-gnu/loongarch64-unknown-linux-gnu/sysroot" \
+  C_INCLUDE_PATH="/usr/loongarch64-unknown-linux-gnu/loongarch64-unknown-linux-gnu/sysroot/usr/include" \
+  LDFLAGS="-L/usr/loongarch64-unknown-linux-gnu/lib/llvm-18/lib"
+
+ADD ./lib/llvm-18 /usr/loongarch64-unknown-linux-gnu/lib/llvm-18
```

Hadolint (2.14.0)

[error] Use COPY instead of ADD for files and folders

(DL3020)